selinux/cockpit_session_selinux.8

```diff
@@ -0,0 +1,255 @@
+.TH  "cockpit_session_selinux"  "8"  "21-04-16" "cockpit_session" "SELinux Policy cockpit_session"
```

Comment:
FTR, it'd be better to integrate that into our normal docbook manpage flow, but this should be a follow-up.